src/components/Label/Label.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable verdaccio/jsx-spread*/
```

Comment:
🤔

Reply:
I was getting an warning and decided to add this `eslint-disable,` but I have just removed it :)

![image](https://user-images.githubusercontent.com/29228205/76160326-7d2ee480-6129-11ea-99a6-398c2dfd1f20.png)